src/modules/search/components/AutocompleteSearch.js

```diff
@@ -150,12 +122,40 @@ export class AutocompleteSearch extends BaElement {
 	 * @override
 	 */
 	createView() {
+		let inputText = this._onSelect.label ? this._onSelect.label : '';
+		const requestData = (e) => {
+			let val = e.target.value;
+			if (!val) {
+				this._clearCandidates();
+				return false;
+			}
+
+			this._searchService.getData(val)
+				.then((data) => {
+					if (data) {
+						this._updateCandidates(data);
+					}
+				});
+		};
+		const onInput = (e) => debounced(200, requestData(e));
+
+		const onClickCandidate = (e) => {
+			const selected = this._candidates[e.target.closest('div').getAttribute('index')];
+			this._onSelect(selected);
+			this._setSearchValue(selected.label);
+			this._clearCandidates();
+		};
+		
 		return html`
 		 <style>${css}</style>
 		 <div class="autocomplete">
-			<input id='autoComplete'/>
+			<input id='autoComplete' .value=${inputText} @input=${onInput}/>
+			${this._candidates ? html`<div id='autocomplete-list' class='autocomplete-items'>${repeat(this._candidates, (candidate) => candidate.id, (candidate, index) => html`
+			<div index=${index} @click=${onClickCandidate} >${unsafeHTML(candidate.labelFormated)}</div>
```

Comment:
Let's try to call `onClickCandidate`  with model (data) of this item

Reply:
OK...done.